etc/bento.env

```diff
@@ -368,7 +368,7 @@ BENTO_PUBLIC_TRANSLATED=true
 BENTO_BEACON_CONTAINER_NAME=${BENTOV2_PREFIX}-beacon
 BENTO_BEACON_NETWORK=${BENTOV2_PREFIX}-beacon-net
 BENTO_BEACON_IMAGE=ghcr.io/bento-platform/bento_beacon
-BENTO_BEACON_VERSION=0.9.0
+BENTO_BEACON_VERSION=v0.9.1
```

Comment:
this shouldnt have a `v` prefix for docker tags (it strips the `v` prefix from github tags)

Reply:
yes, I discovered this the hard way on QA